src/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"; 
+import './Nav.css';
+
+import {
+    Link, 
+} from "react-router-dom";
+
+import {
+    Navbar,
+    NavbarBrand,
+} from "reactstrap";
+
+
+const Nav = (props) => {
+    return(
+    <div>
+        <Navbar color="light">
+            <span className="navbar-brand">
+                <Link className="brand-text" to="/">NVolve2</Link>
+            </span>
+        </Navbar>
+    </div>);
```

Comment:
Put this close paren on her own line please. She is a strong independent woman and she don't need no div.